core/templates/dev/head/pages/exploration_player/SupplementalCardDirective.js

```diff
@@ -62,6 +69,43 @@ oppia.directive('supplementalCard', [
             $scope.helpCardHasContinueButton = false;
           };
 
+          $scope.showHint = function() {
+            var hints = $scope.currentInteractionHints;
+            var numHintsConsumed = hintManagerService.getNumHintsConsumed();
+            if (numHintsConsumed < hints.length) {
+              var currentHint = hints[numHintsConsumed].hintText;
+              playerTranscriptService.addNewAnswer(
```

Comment:
As above -- I don't think you should treat this as answer/feedback. Hint requests may look similar in the UI but conceptually they are different. There should also not be any strings along the lines of "I would like a hint." in the JS -- they should be in the template so that they can be internationalized.